src/index.test.ts

```diff
@@ -0,0 +1,5 @@
+import { add } from "./index";
+
+test("adds two numbers", () => {
```

Comment:
we prefer `test` over `describe`/`it` structure?

Reply:
My personal preference is to add `describe` only once it becomes useful to group tests. For a test file containing a single test, it feels unnecessary to me.

However an argument could be made to standardize early and always on `describe` at the root-level.

As for `it` vs `test`, I marginally prefer `test` because it states more clearly _what it is_. Whereas the motivation for calling tests `it` is as far as I can tell, solely based on the cute wordplay that allows you to read the `it` as the beginning of a sentence continued by the test name. That can be a useful remainder of how to name tests well, but it doesn't compel me to prefer `it` to `test`. _That said_ this is all very minor and subjective. Happy to switch to `it` if we'd rather just stick to existing conventions in say lokalise-main (and/or elsewhere in the organization).

Reply:
In this case `describe/it` is an established convention in our org, which, while not any better than `test` one, we don't seem to have a strong reason to change.

Reply:
OK, agreed in the name of consistency.